app/models/user.rb

```diff
@@ -92,7 +92,7 @@ def password_complexity
 
   def associate_points_of_contact
     if Settings.current.auto_associate_pocs
-      Vendor.where('points_of_contact.email' => email).each do |vendor|
+      Vendor.where('points_of_contact.email' => email).by_updated_at.each do |vendor|
```

Comment:
This seems like it is another case of not needing the `by_updated_at` since I don't believe the order of adding a role matters.